### ISSUE TITLE
Error "list index out of range"

### ISSUE THREAD

**Issue by MarkusBe** [ptrstn/dailyblink#1]
Hello, since a couple of weeks I am getting an error saying "list index out of range".

Any Ideas?

**Comment by ptrstn** [comment]
Cloudfare seems to block the request and returns a 403 response code.

I am not quite sure how to fix this yet, but will look into this.

**Comment by MarkusBe** [comment]
Thanks for the quick response. I really like this script, would be great if you could find a solution for this issue. Thanks for your efforts...

**Comment by ptrstn** [comment]
Should be fixed now in version ```0.5.1```. Let me know if it works.

```bash
pip install --user --upgrade git+https://github.com/ptrstn/dailyblink
pip install --user --upgrade git+https://github.com/ptrstn/dailyblink
```

```
Downloading the free daily Blinks...

english (en):
168 Hours - Laura Vanderkam

Saving book text...
Saving book cover...
Saving audio track #1 - What’s in it for me? Learn how to fit ev...
Saving audio track #2 - You’re not too busy – you’re just not in...
Saving audio track #3 - To live a fulfilling life, focus on your...
Saving audio track #4 - Choosing a job that uses your core compe...
Saving audio track #5 - To become time-rich you must control you...
Saving audio track #6 - To achieve a career breakthrough, combin...
Saving audio track #7 - You can improve your relationships with ...
Saving audio track #8 - Outsource housework that doesn’t align w...
Saving audio track #9 - Plan your leisure time to increase its v...
Saving audio track #10 - Final summary...

german (de):
Chronik einer angekündigten Krise - Paul Schreyer

Saving book text...
Saving book cover...
Saving audio track #1 - Was drin ist für dich: Ein alternativer ...
Saving audio track #2 - Manche wissenschaftlichen Erkenntnisse s...
Saving audio track #3 - Das Forschungsfeld der Biosecurity ist e...
Saving audio track #4 - Seit 2001 führt die Johns Hopkins Univer...
Saving audio track #5 - Ende 2019 wurde das Drehbuch für eine in...
Saving audio track #6 - Vertreter der Population Control bereite...
Saving audio track #7 - Anfang 2020 wurde das Corona-Drehbuch ve...
Saving audio track #8 - Politik und Medien verbreiteten während ...
Saving audio track #9 - Wir sollten uns trauen, das Coronavirus ...
Saving audio track #10 - Fazit...

All blinks were saved under /home/ptr/Musik/blinks
```

**Comment by MarkusBe** [comment]
Yes, it works again, thank you!
